_maps/map_files/RandomRuins/SpaceRuins/dj.dmm

```diff
@@ -516,25 +568,52 @@
 /turf/simulated/floor/plasteel/dark,
 /area/ruin/space/djstation)
 "bq" = (
-/obj/structure/table,
-/obj/item/radio/intercom/pirate,
+/obj/machinery/atmospherics/pipe/simple/hidden{
+	dir = 4
+	},
 /turf/simulated/floor/plasteel{
-	dir = 4;
+	dir = 6;
 	icon_state = "darkblue"
 	},
 /area/ruin/space/djstation)
 "br" = (
-/obj/machinery/door/airlock/public/glass{
-	name = "Cabin"
+/obj/effect/spawner/window/shuttle,
+/obj/structure/cable{
+	d2 = 4;
+	icon_state = "0-4"
 	},
-/turf/simulated/floor/plasteel/dark,
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 4;
+	icon_state = "1-4"
+	},
+/obj/structure/cable{
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4"
+	},
+/turf/simulated/floor/plating,
 /area/ruin/space/djstation)
 "bs" = (
-/turf/simulated/floor/plasteel/grimy,
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/item/gps/ruin{
+	gpstag = "Soviet Entertainment Broadcast";
+	pixel_x = -32;
+	pixel_y = 32
+	},
```

Comment:
Any reason why it needs this extreme positioning?

Reply:
> Any reason why it needs this extreme positioning?

The GPS console is on the floor and pixel shifted into the wall at a wall corner.

Reply:
And you cant place next to it because of a locker right?

Reply:
Because of the window directly below the wall, it would be possible for the console to magically disappear from view if a player looked from outside the kitchen window and moved vertically from bottom to top.